docs/resources/cse_microservice_engine.md

```diff
@@ -90,6 +90,10 @@ In addition to all arguments above, the following attributes are exported:
 
 * `instance_limit` - The maximum number of the microservice instance resources.
 
+* `service_registry_address` - The connection address of service center.
```

Comment:
do we need to add both public and private address here?

Reply:
Actually we only focuses on public addresses

Reply:
but I find you set this to serviceEndpoint if public address not set. what does serviceEndpoint mean?

Reply:
I think adding a note about this is better.

Reply:
If the public network IP is not enabled, the address set here is the address used for intranet connection.
If it is enabled, the external network is set.

Reply:
If there is a public network, the microservice connect to the public network.
If there is no public network, the microservice connect to the internal network.

Reply:
new structure is support.